src/cardsavr/CardsavrJSLibrary-2.0.ts

```diff
@@ -66,6 +66,10 @@ export class CardsavrSession {
         };
     };
 
+    getSessionUserId() : number {
+        return +this._sessionData.userId;
```

Comment:
+?

Reply:
I am now storing the userId so I don't have to keep looking it up.  I have been looking it up on every job post.